packages/web/src/scss/components/Modal/README.md

```diff
@@ -334,58 +322,7 @@ When you put it all together:
 <!-- Modal: end -->
 ```
 
-## Simple Modal (DEPRECATED)
-
-A simple version of Modal.
-
-⚠️ **DEPRECATED:** Simple Modal will be removed in the next major version in favour
-of the composed Modal above.
-
-Example usage:
-
-```html
-<button
-  type="button"
-  class="Button Button--primary Button--medium"
-  data-toggle="modal"
-  data-target="#modal-example-1"
-  aria-controls="modal-example-1"
-  aria-expanded="false"
->
-  Open Modal
-</button>
-
-<dialog id="modal-example-1" class="Modal">
-  <div class="Modal__content">
-    <div class="Modal__dialog">
-      <div class="Modal__header">
-        <button
-          type="button"
-          class="Button Button--tertiary Button--medium Button--square"
-          data-dismiss="modal"
-          data-target="#modal-example-1"
-          aria-controls="modal-example-1"
-          aria-expanded="false"
-        >
-          <svg class="Icon" width="24" height="24" aria-hidden="true">
-            <use xlink:href="/icons/svg/sprite.svg#close" />
-          </svg>
-          <span class="accessibility-hidden">Close</span>
-        </button>
-      </div>
-      <div class="Modal__body">
-        <p>
-          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aliquam at excepturi laudantium magnam mollitia
-          perferendis reprehenderit, voluptate. Cum delectus dicta ducimus eligendi excepturi natus perferendis
-          provident unde. Eveniet, iste, molestiae?
-        </p>
-      </div>
-    </div>
-  </div>
-</dialog>
-```
-
-## Toggle Attributes
+# Toggle Attributes
```

Comment:
I am sorry if I said it confusingly, the heading structure should be moved only in the Modal/ModalComposed section. These headings shouldn't be H1 (only one #). 

Please this heading and the next ones in this file should be ##

```suggestion
## Toggle Attributes
```

Reply:
@crishpeen You wrote "I agree we can remove this heading and then we can increase the level of the rest of the headings." And that's exactly what I did

Reply:
fixed